locales/en.json

```
@@ -249,6 +249,9 @@
       },
       "feeEstimate": {
         "title": "Fee estimate"
+      },
+      "outgoingChannel": {
+        "title": "Select Outbound Channel"
```

I think we can skip "Select" to keep it shorter 

```suggestion
        "title": "Outbound Channel"
```